.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,111 @@
+name: "Bug Report"
+description: Report a bug with the Kubecost Helm chart. Please read the issue guidelines to determine what is an appropriate bug report for this repository.
```

Comment:
Is there a user-friendly way to link to guidelines here or with other refs?

Reply:
Yes, added relative link.